src/main/java/org/opencord/kafka/integrations/AaaKafkaIntegration.java

```diff
@@ -118,12 +161,38 @@ private JsonNode serialize(AuthenticationEvent event) {
         return authEvent;
     }
 
+    private JsonNode serializeStat(AuthenticationStatisticsEvent event) {
+        log.info("Serializing AuthenticationStatisticsEvent");
+        ObjectMapper mapper = new ObjectMapper();
+        ObjectNode authMetricsEvent = mapper.createObjectNode();
+        authMetricsEvent.put(TIMESTAMP, Instant.now().toString());
+        authMetricsEvent.put(ACCEPT_PACKET_COUNTER, event.subject().getAcceptPacketsCounter().get());
```

Comment:
Modify all getXXX() methods to return the long value and not the AtomicLong instances